src/lib.rs

```diff
@@ -766,8 +787,22 @@ mod tests {
 
     #[test]
     fn test_from_bytes() {
-        let b = vec!( 0xa1, 0xa2, 0xa3, 0xa4, 0xb1, 0xb2, 0xc1, 0xc2,
-                   0xd1, 0xd2, 0xd3, 0xd4, 0xd5, 0xd6, 0xd7, 0xd8 );
+        let b = vec!(0xa1,
+                     0xa2,
+                     0xa3,
+                     0xa4,
+                     0xb1,
+                     0xb2,
+                     0xc1,
+                     0xc2,
+                     0xd1,
+                     0xd2,
+                     0xd3,
+                     0xd4,
+                     0xd5,
+                     0xd6,
+                     0xd7,
+                     0xd8);
```

Comment:
I recommend using `#[rustfmt_skip]` for this sort of thing. They tend to be rare and hard for rustfmt to detect in general, so it is unlikely we'll be able to fix it in the short term.